agent/05_agent_configure.sh

```diff
@@ -407,6 +428,15 @@ else
   configure_dnsmasq ${ip} ""
 fi
 
+# Note that BOND_PRIMARY_INTERFACE creates the same mac on both ports so can't be used here
```

Comment:
I would love to apply this logic to the existing feature. While the duplicate macs work, it's obviously less than ideal. Would you have any interest in just updating BOND_PRIMARY_INTERFACE to use this method?

Reply:
> I would love to apply this logic to the existing feature. While the duplicate macs work, it's obviously less than ideal. Would you have any interest in just updating BOND_PRIMARY_INTERFACE to use this method?

@cybertron Yes I'd love to make this common and just use one variable. Let me try moving it out of agent. I would eventually need some help to test it in an non-agent setup.

One thing regarding the naming - since the env variable can be set the bonding mode I'm not sure if `BOND_PRIMARY_INTERFACE` is the right name for the setting.

Reply:
@cybertron take a look at the latest changes and see if that makes sense,